### PR TITLE
tmuxPlugins.dracula: init at unstable-2020-12-2

### PR DESCRIPTION
**Pull request by ethancedwards8**
###### Motivation for this change

Adding the tmux plugin Dracula, it is a theme among other things like cpu usage, ram usage, and battery percentage.

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [x] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by ethancedwards8** [comment]
Hmmm I don't think everything should be rebuilding like this?

**Comment by SuperSandro2000** [comment]
Result of `nixpkgs-review pr 104652` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>tmuxPlugins.dracula</li>
  </ul>
</details>

**Comment by SuperSandro2000** [comment]
> Hmmm I don't think everything should be rebuilding like this?

For me there is just one rebuild which is expected when adding a new plugin. What do you think is going wrong here exactly?

**Comment by ethancedwards8** [comment]
I'm actually not sure then, it just looked like a lot of packages were building when I ran my tests locally. I guess it all checks out. @SuperSandro2000

File: pkgs/misc/tmux-plugins/default.nix
Comment (buckley310) [inline pkgs/misc/tmux-plugins/default.nix:139]: Indentation here is slightly off

**Comment by SuperSandro2000** [comment]
> I'm actually not sure then, it just looked like a lot of packages were building when I ran my tests locally. I guess it all checks out. @SuperSandro2000 

They probably where build because you didn't have them locally and they where not available from cache. What counts is what needs to be built before and after the PR.

File: pkgs/misc/tmux-plugins/default.nix
Comment (ethancedwards8) [reply pkgs/misc/tmux-plugins/default.nix]: Fixed. 👍

**Comment by ethancedwards8** [comment]
> > I'm actually not sure then, it just looked like a lot of packages were building when I ran my tests locally. I guess it all checks out. @SuperSandro2000
> 
> They probably where build because you didn't have them locally and they where not available from cache. What counts is what needs to be built before and after the PR.

Okay makes sense.

**Comment by ethancedwards8** [comment]
Also, this is my first nixpkgs PR so if I'm doing anything else wrong let me know and I'll be sure to fix it.

**Comment by buckley310** [comment]
Squashing all the commits into a single commit is generally preferred.
Also for consistency you probably ought to rename the PR to be the same as the commit message.

**Comment by ethancedwards8** [comment]
Ahh, I've never squashed anything before, and I'm failing miserably at it.

**Comment by ethancedwards8** [comment]
It seems I've caused a boo boo, not knowing how to properly squash commits...

File: pkgs/misc/tmux-plugins/default.nix
Comment (timstott) [inline pkgs/misc/tmux-plugins/default.nix:138]: :books: Would it be possible to add the meta attributes, you can look at [resurrect](https://github.com/NixOS/nixpkgs/blob/440c5808a18980909ca2eb349d4819d266967e49/pkgs/misc/tmux-plugins/default.nix#L294-L319) as an example (omit `longDescription` if unavailable).

You will also need to add yourself as a maintainer of this package. Add yourself to the maintainer list which you can find [here](https://github.com/NixOS/nixpkgs/blob/master/maintainers/maintainer-list.nix). It should be updated in its own commit with the message `maintainers: add <username>`.

You can find your `githubId` with `curl -s https://api.github.com/users/kyleondy | jq '.id'`

Comment (ethancedwards8) [reply pkgs/misc/tmux-plugins/default.nix:138]: Done 👍

**Comment by ethancedwards8** [comment]
And now I rebase...

**Comment by ethancedwards8** [comment]
Thanks for the help, this has definitely taught me a lot about git and Nix.

**Review by timstott — APPROVED** [review]
Thank you for your contribution @ethancedwards8, this is excellent :tophat:

**Comment by ethancedwards8** [comment]
Ah, didn't notice there was already a PR, whoops. Since I'm a maintainer of the actual package I feel it's right I package it :)

File: pkgs/misc/tmux-plugins/default.nix
Comment (SuperSandro2000) [inline pkgs/misc/tmux-plugins/default.nix:146]:
```suggestion
    meta = with stdenv.lib; {
      homepage = "https://draculatheme.com/tmux";
      description = "A feature packaed Dracula theme for tmux!";
      license = licenses.mit;
      platforms = platforms.unix;
      maintainers = with maintainers; [ ethancedwards8 ];
    };
  };
```

But won't block the merge.

File: pkgs/misc/tmux-plugins/default.nix
Comment (SuperSandro2000) [inline pkgs/misc/tmux-plugins/default.nix:136]: But this will. Please either tag the releases in your repo or switch the version to ``unstable-year-month-date``.

**Comment by SuperSandro2000** [comment]
Result of `nixpkgs-review pr 104652` run on x86_64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>tmuxPlugins.dracula</li>
  </ul>
</details>

**Comment by SuperSandro2000** [comment]
Result of `nixpkgs-review pr 104652` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>tmuxPlugins.dracula</li>
  </ul>
</details>

File: pkgs/misc/tmux-plugins/default.nix
Comment (ethancedwards8) [reply pkgs/misc/tmux-plugins/default.nix]: Done. :+1:

**Comment by SuperSandro2000** [comment]
Result of `nixpkgs-review pr 104652` run on x86_64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>tmuxPlugins.dracula</li>
  </ul>
</details>

**Comment by SuperSandro2000** [comment]
Result of `nixpkgs-review pr 104652` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>tmuxPlugins.dracula</li>
  </ul>
</details>

**Pull request merged**
